django/contrib/admin/templates/admin/change_list.html

```diff
@@ -73,7 +73,7 @@
       </div>
       {% block filters %}
         {% if cl.has_filters %}
-          <nav id="changelist-filter" aria-labelledby="changelist-filter-header">
```

Comment:
We also have `<form id="changelist-search" method="get" role="search">` in `django/contrib/admin/templates/admin/search_form.html` should we add an `aria-label` here as described in [labelling landmarks](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Search_role#labeling_landmarks)?

Reply:
We should yes. For some reason I thought we had this already but it looks like I was looking at the wrong thing.

Reply:
It's better to use a heading and `aria-labelledby`, because:

1. Screen reader users can navigate using headings.
2. `aria-label` doesn't get translated by translation tools like (by not limited to) Google Translate.